src/SprykerEco/Zed/AmazonQuicksight/Business/AmazonQuicksightFacadeInterface.php

```diff
@@ -46,7 +50,28 @@ public function expandUserCollectionWithQuicksightUsers(
      *
      * @return \Generated\Shared\Transfer\UserCollectionResponseTransfer
      */
-    public function createQuicksightUsersForUserCollectionResponse(
+    public function createQuicksightUsersByUserCollectionResponse(
+        UserCollectionResponseTransfer $userCollectionResponseTransfer
+    ): UserCollectionResponseTransfer;
+
+    /**
+     * Specification:
+     * - Filters out users with statuses not applicable for deleting quicksight user.
+     * - Uses {@link \SprykerEco\Zed\AmazonQuicksight\AmazonQuicksightConfig::getUserStatusesApplicableForQuicksightUserDeletion()} to get a list of user statuses applicable for deleting quicksight user.
+     * - Filters out users without persisted quicksight user.
+     * - Sends request to AWS API to delete Quicksight users. For more information see {@link https://docs.aws.amazon.com/quicksight/latest/APIReference/API_DeleteUser.html}.
+     * - If AWS API call returns error it is added to `QuicksightUserCollectionResponseTransfer.errors` with `idQuicksightUser` as entity identifier.
+     * - If AWS API call returns error, quicksight user will not be deleted from persistence.
```

Comment:
```suggestion
     * - If AWS API call returns error, Quicksight user will not be deleted from persistence.
```
please fix for all the specs